pkg/ccl/backupccl/system_schema.go

```diff
@@ -35,6 +44,92 @@ const (
 //    cluster so there is no need to rewrite system table data.
 type systemBackupConfiguration struct {
 	includeInClusterBackup clusterBackupInclusion
+	// customRestoreFunc is responsible for restoring the data from a table that
+	// holds the restore system table data into the given system table. If none
+	// is provided then `defaultRestoreFunc` is used.
+	//
+	// TODO: Do we need to pass in the system table name here?
```

Comment:
Was going to comment below that seemed like the table-specific func could just be expected to know its name

Reply:
I like that the custom restore func keeps the same signature as the default restore func, and I think the default restore func needs to take in the name since it's generic for multiple system tables.

Reply:
works for me